### ISSUE TITLE
Not accurate

### ISSUE THREAD

**Issue by emretekince** [GantMan/jail-monkey#20]
Hi,
My users say that their device are not rooted but they still got blocked on app because jail-monkey see them as rooted. Is there a bug?

**Comment by GantMan** [comment]
Eeek, can you report version/platform so I can check it?

**Comment by die20** [comment]
Is this really a problem with this plugin? I’m considering using it but I don’t want it to throw false alarms.

**Comment by GantMan** [comment]
I no-longer have a rooted iphone to test with right now.   I will see about getting one, but that's the hardest part to testing this.

PRs welcome, and I can help anyone needing help with the code.

**Comment by die20** [comment]
Cool! Thanks!

Even if the rooted method is flawed (which it might not be), using react-native-device-info for checking if the device is running as an emulator would solve that. 

How confident are you that the jail broken test is reliable?

**Comment by GantMan** [comment]
I've asked a friend of mine to mail me a jail broken phone.  Any testing you have would be awesome, but this was well tested when I first did the library... no clue on recent.

New testing would be good.

**Comment by jerodgers1** [comment]
Any update on the testing?

**Comment by GantMan** [comment]
No.  I decided to put a large warning on the the README because I can't test.

**Comment by RohitPatilAtShell** [comment]
I am also getting this issue in one of my client device:
Brand: Samsung
OS: Lineageos-18 
Android: 11

I have already tested that the device is not rooted(attached screenshot).
Its a false alarm!!!

Please help me to get this issue resoled.
Thanks in advance.


![MicrosoftTeams-image (6)](https://user-images.githubusercontent.com/92922605/202458726-6d8315fa-78a7-41b9-a88c-625e6c009540.png)
